src/style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.FirstForm {
+  margin: 20px 8px -3px;
+}
+
+.search-1 {
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  padding-left: 62px;
+  padding-bottom: 2px;
+
+  background: #fff;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Comment:
[checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 2nd item, move similar values to `search` class